OpenTESArena/src/Entities/Camera3D.cpp

```diff
@@ -6,8 +6,8 @@
 #include "../Math/Constants.h"
 #include "../Math/Quaternion.h"
 
-Camera3D::Camera3D(const Double3 &position, const Double3 &direction)
-	: position(position), forward(direction)
+Camera3D::Camera3D(const Double3 &direction, const Double3 &position)
```

Comment:
Yeah. I might tweak it some in the future depending on how I want to use Camera3D.